src/components/projects/CostaRica.jsx

```diff
@@ -1,5 +1,88 @@
+import Subtitle from "../Subtitle";
 const CostaRica = () => {
-  return <div>CostaRica</div>;
+  return (
+    <div className="grid grid-cols-3 gap-10 m-28">
+      <div className="col-span-2 pb-4">
+        <Subtitle title="General Overview"></Subtitle>
+        <p className="ml-2 mt-1">
+          In collaboration with Claudia Chaves Villarreal, a Ph.D. Candidate and
+          her organization Biogas Pococi, we are working to implement
+          Biodigestor tanks and micro controlling systems. She has already
+          implemented this system for 38 families! The Biodigestor provides
+          clean fuel without the emission of harmful particulates that has had
+          detrimental impacts on the health of rural inhabitants. The
+          Biodigestor also provides a source of fertilizer for farmers to use.
+        </p>
+      </div>
+      <div>
+        <img
+          src="/images/CostaRica/CostaRica1.gif"
+          alt="Costa Rica Image 1"
+        ></img>
+      </div>
+
+      <div className="col-span-2 pb-4">
+        <Subtitle
+          title="Motivation"
+          description="Rural communities are already disadvantaged socioeconomically"
+        ></Subtitle>
+        <p className="ml-2 mt-1">
+          <ul
+            className="marker:text-ewb-blue-200 ml-4"
+            style={{ listStyleType: "square" }}
+          >
+            <li>
+              Include them in the building process by teaching them the basics
+              of engineering
+            </li>
+            <li>Inspire them to peruse education</li>
+            <li>Improve infrastructure of land and quality of life</li>
+          </ul>
+        </p>
+      </div>
+      <div>
+        <img
+          src="/images/CostaRica/CostaRica2.gif"
+          alt="Costa Rica Image 2"
+        ></img>
+      </div>
+
+      <div className="col-span-3 pb-4">
+        <Subtitle
+          title="Educational Micro Controller Projects"
+          description="Education is very hard to obtain affecting isolated rural communities the most"
+        ></Subtitle>
+        <p className="ml-2 mt-1">
+          <ul
+            className="marker:text-ewb-blue-200 ml-4"
+            style={{ listStyleType: "square" }}
+          >
+            <li>
+              Providing some micro controllers and electronic parts allow for
+              community members, from kids to adults, gain a valuable skill that
+              improves their future opportunities
+            </li>
+            <li>
+              Is beneficial for both the volunteer(you), to gain once a lifetime
+              experience, and to the disadvantage community by improving their
+              way of life
+            </li>
+            <li>
+              {" "}
+              A water treatment system will also be implemented, to learn more
+              check out the{" "}
```

Comment:
{" "}

Reply:
![image](https://github.com/acm-ucr/ewb-website/assets/135681592/62d1fce1-717e-4fc6-938c-03c8df264aed)
keeping this {" "} otherwise it'll look like this